BungeeCord-Patches/0046-Make-InitialHandler-connections-logging-optional-dis.patch

```diff
@@ -0,0 +1,77 @@
+From d8b5c0e6f8a30d61669ae4cd7bd4a675d6bfe1a9 Mon Sep 17 00:00:00 2001
+From: Gabriele C <sgdc3.mail@gmail.com>
+Date: Thu, 8 Feb 2018 19:10:52 +0100
+Subject: [PATCH] Make InitialHandler connections logging optional (disabled by
+ default)
+
+
+diff --git a/api/src/main/java/net/md_5/bungee/api/ProxyConfig.java b/api/src/main/java/net/md_5/bungee/api/ProxyConfig.java
+index 43519ce1..2631d6bb 100644
+--- a/api/src/main/java/net/md_5/bungee/api/ProxyConfig.java
++++ b/api/src/main/java/net/md_5/bungee/api/ProxyConfig.java
+@@ -168,6 +168,11 @@ public interface ProxyConfig
+      */
+     boolean isLogServerListPing();
+ 
++    /**
++     * Whether we log InitialHandler connections
++     */
++    boolean isLogInitialHandlerConnections();
++
+     /**
+      * The supported versions
+      */
+diff --git a/proxy/src/main/java/io/github/waterfallmc/waterfall/conf/WaterfallConfiguration.java b/proxy/src/main/java/io/github/waterfallmc/waterfall/conf/WaterfallConfiguration.java
+index 7dc3c2d9..6dc58ca2 100644
+--- a/proxy/src/main/java/io/github/waterfallmc/waterfall/conf/WaterfallConfiguration.java
++++ b/proxy/src/main/java/io/github/waterfallmc/waterfall/conf/WaterfallConfiguration.java
+@@ -26,6 +26,13 @@ public class WaterfallConfiguration extends Configuration {
+      */
+     private boolean logServerListPing = false;
+ 
++    /**
++     * Whether we log InitialHandler connections
++     * <p>
++     * Default is false (don't log)
++     */
++    private boolean logInitialHandlerConnections = false;
```

Comment:
Default to true, don't disable logging messages which provide some level of information as to what is going on

Reply:
This log message tells you only one information:
Not under attack<- (YES) Can you read the console log? (NO) -> Under attack
😆

Reply:
It makes sense only in debug environments

Reply:
It's a message which can provide information on what is going on, which is the reason why the message is there in the first place. I don't agree with disabling information messages by default which can be useful should something be going on, just because you disagree with that notion.